pkg/controller/storagecluster/reconcile.go

```diff
@@ -120,20 +143,14 @@ func (r *ReconcileStorageCluster) Reconcile(request reconcile.Request) (reconcil
 		}
 		if !isActive {
 			instance.Status.Phase = statusutil.PhaseIgnored
-			phaseErr := r.client.Status().Update(context.TODO(), instance)
-			if phaseErr != nil {
-				reqLogger.Info("Status Update Error", "StatusUpdateErr", "Failed to set PhaseProgressing")
-				return reconcile.Result{}, phaseErr
-			}
 			return reconcile.Result{}, nil
 		}
 	} else if instance.Status.Phase == statusutil.PhaseIgnored {
 		return reconcile.Result{}, nil
 	}
 
 	if !instance.Spec.ExternalStorage.Enable {
-		err = r.validateStorageDeviceSets(instance)
-		if err != nil {
+		if err := r.validateStorageDeviceSets(instance); err != nil {
```

Comment:
Probably belongs in the same other commit as the variable rename

Reply:
done